index.bs

```diff
@@ -2449,6 +2442,13 @@ To <dfn>obtain and deliver a debug report on source registration</dfn> given a
     : "<code>[=source debug data type/source-storage-limit=]</code>"
     :: [=map/Set=] |body|["`limit`"] to the user agent's [=max pending sources per source origin=],
          [=serialize an integer|serialized=].
+    : "<code>[=source debug data type/source-channel-capacity-limit=]</code>"
+    :: [=map/Set=] |body|["`limit`"] to the user agent's [=max event-level channel capacity per source=],
+         [=serialize an integer|serialized=].
```

Comment:
The max here is a double, not an integer.

Reply:
thanks, I messed the two variables up. Fixed, thanks.